tests/webidl/output_ALL.txt

```diff
@@ -109,5 +109,6 @@ Assertion failed: [CHECK FAILED] Parent::Parent(val:val): Expecting <integer>
 Parent:42
 Assertion failed: [CHECK FAILED] Parent::voidStar(something:something): Expecting <pointer>
 Assertion failed: [CHECK FAILED] StringUser::Print(anotherString:anotherString): Expecting <string>
+true
```

Comment:
strcmp returns zero if the strings match so wouldn't we expect to see `false` here?

Reply:
You are right, I needed to be passing a pointer to memory I had allocated to get the correct result.

Reply:
That would be correct, I needed to malloc some memory beforehand to get the test to work properly.